grakn-test/pom.xml

```diff
@@ -198,6 +197,23 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19.1</version>
+                <configuration>
+                    <systemProperties>
+                        <logback.configurationFile>${main.basedir}/conf/test/logback-test.xml</logback.configurationFile>
```

Comment:
Could it be using the wrong `${main.basedir}`? Maybe the base dir of the test package instead of the room folder?